code/modules/projectiles/projectile.dm

```diff
@@ -102,6 +105,11 @@ var/list/impact_master = list()
 	var/fire_sound = 'sound/weapons/Gunshot.ogg' //sound that plays when the projectile is fired
 	var/rotate = 1 //whether the projectile is rotated based on angle or not
 
+/obj/item/projectile/New()
+	..()
+	initial_pixel_x = pixel_x
```

Comment:
@clusterfack told me that using initial() was bad. And since we need that value at every step of the projectile's path, I'd rather store it on a variable first instead.